e2e-tests/functions

```diff
@@ -389,7 +393,7 @@ wait_job_completion() {
 		fi
 	done
 	echo
-	set -o xtrace
+	# set -o xtrace
```

Comment:
why do we need to have it?

Reply:
We don't. It was temporary for debugging purposes. Thanks for catching this!